plugins/mission/CMakeLists.txt

```diff
@@ -5,8 +5,24 @@ add_library(dronecore_mission SHARED
     mission_item_impl.cpp
 )
 
+include_directories(
+    ${CMAKE_SOURCE_DIR}/core
+    SYSTEM ${CMAKE_SOURCE_DIR}/libs/include
+    SYSTEM libs/json11
+)
+
+# JSON parser library for parsing QGC plan for Mission
+add_subdirectory(libs/json11)
+
+if(ANDROID)
```

Comment:
Some of the things here like this Android hack are no longer required.

Reply:
OK. I will delete them.